samcli/local/docker/lambda_image.py

```diff
@@ -114,7 +115,8 @@ def build(self, runtime, packagetype, image, layers, stream=None):
         # Default image tag to be the base image with a tag of 'rapid' instead of latest.
         # If the image name had a digest, removing the @ so that a valid image name can be constructed
         # to use for the local invoke image name.
-        image_tag = f"{image_name.split(':')[0].replace('@', '')}:rapid-{version}"
+        image_repo = image_name.split(":")[0].replace("@", "")
+        image_tag = f"{image_repo}:rapid-{version}"
```

Comment:
More a nit, than anything. Lets not refer to "rapid" in static string anywhere directly, what do you think about making it a constant that we can refer to in f strings?

Reply:
Other than that, looks good!